### ISSUE TITLE
Adding TorchScript test for `F.phase_vocoder`

### ISSUE THREAD

**Issue by mthrok** [pytorch/audio#1377]
[`torchaudio.functional.phase_vocoder`](https://github.com/pytorch/audio/blob/ea857940de9e3738166989ad3bf1726741a13f04/torchaudio/functional/functional.py#L558) does not have TorchScript-compatibility test. We need this before adding native complex dtype support.

## Steps
1. Add `test_phase_vocoder` method in [`torchscript_consistency_impl.py`](https://github.com/pytorch/audio/blob/ea857940de9e3738166989ad3bf1726741a13f04/test/torchaudio_unittest/functional/torchscript_consistency_impl.py)
    Follow how the other functions are tested. It should be straightforward but if you need help feel free to ask here.

## Build & testing
For setting up dev env, please refer to [CONTRIBUTING.md](https://github.com/pytorch/audio/blob/master/CONTRIBUTING.md).
Run the test with

```bash
(cd test && pytest torchaudio_unittest/functional/torchscript_consistency_cpu_test.py -k phase)
```

**Comment by mthrok** [comment]
#1379

**Comment by mthrok** [comment]
resolved by #1379
